release-notes/7.0/preview/api-diff/preview6/Microsoft.WindowsDesktop.App/7.0-preview6_System.Windows.Forms.md

```diff
@@ -0,0 +1,485 @@
+         }
+     }
+     [DefaultEventAttribute("Scroll")]
+     [DefaultPropertyAttribute("Value")]
+     public abstract class ScrollBar : Control {
+-        [BindableAttribute(false)]
+-        [BrowsableAttribute(false)]
+-        [DesignerSerializationVisibilityAttribute(0)]
+-        [EditorBrowsableAttribute(1)]
+-        public override string Text { get; set; }
++        [BindableAttribute(false)]
++        [BrowsableAttribute(false)]
++        [DesignerSerializationVisibilityAttribute(0)]
++        [EditorBrowsableAttribute(1)]
++        [AllowNullAttribute]
++        public override string Text { get; set; }
+     }
+     public class SelectionRangeConverter : TypeConverter {
+-        public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContext? context, object value, Attribute[]? attributes);
++        [RequiresUnreferencedCodeAttribute("The Type of value cannot be statically discovered. The public parameterless constructor or the 'Default' static field may be trimmed from the Attribute's Type.")]
++        public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContext? context, object value, Attribute[]? attributes);
+     }
+     [DesignerAttribute("System.Windows.Forms.Design.SplitterPanelDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+     [DockingAttribute(DockingBehavior.Never)]
+     [ToolboxItemAttribute(false)]
+     public sealed class SplitterPanel : Panel {
+-        [BrowsableAttribute(false)]
+-        [DesignerSerializationVisibilityAttribute(0)]
+-        [EditorBrowsableAttribute(1)]
+-        public new string Name { get; set; }
++        [BrowsableAttribute(false)]
++        [DesignerSerializationVisibilityAttribute(0)]
++        [EditorBrowsableAttribute(1)]
++        [AllowNullAttribute]
++        public new string Name { get; set; }
+     }
+     [DefaultPropertyAttribute("ColumnCount")]
+     [DesignerAttribute("System.Windows.Forms.Design.TableLayoutPanelDesigner, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+     [DesignerSerializerAttribute("System.Windows.Forms.Design.TableLayoutPanelCodeDomSerializer, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", "System.ComponentModel.Design.Serialization.CodeDomSerializer, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+     [DockingAttribute(DockingBehavior.Never)]
+     [ProvidePropertyAttribute("CellPosition", typeof(Control))]
+     [ProvidePropertyAttribute("Column", typeof(Control))]
+     [ProvidePropertyAttribute("ColumnSpan", typeof(Control))]
+     [ProvidePropertyAttribute("Row", typeof(Control))]
+     [ProvidePropertyAttribute("RowSpan", typeof(Control))]
+     public class TableLayoutPanel : Panel, IExtenderProvider {
```

Comment:
@RussKie @JeremyKuhne is it worth showing any of these attributes? Let me know if I should ignore any of them from now on.

Reply:
I don't think it's worth showing any attributes in a diff unless those are changed (i.e., added/updated/removed).